engine/chain/internal/bhs/get_merkleroots_from_bhs_test.go

```diff
@@ -0,0 +1,108 @@
+package bhs_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/chain"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetMerkleRootsFromBHSSuccess(t *testing.T) {
+	t.Run("Get MerkleRoots success", func(t *testing.T) {
```

Comment:
Because it is used only for kinda proxy or reading, I would prefer to add `integration`/`endpoint` tests instead unit tests. 

If you want to make this change in a separate PR. So it wouldn't block this PR.

Reply:
okay, lets make it in a separate pr